packages/@aws-cdk/aws-codepipeline/lib/cross-region-support-stack.ts

```diff
@@ -71,6 +71,8 @@ export interface CrossRegionSupportStackProps {
    * @example '012345678901'
    */
   readonly account: string;
+
+  readonly synthesizer: cdk.IStackSynthesizer | undefined;
```

Comment:
Using a `?` is the more usual way we express that.

Reply:
Yes, but I actually wanted to make the `synthesizer` a required property.

I don't think it matters much, this is all private to the `codepipeline` module anyway.